JsonClassGeneratorLib/CodeWriters/CSharpCodeWriter.cs

```diff
@@ -323,9 +333,17 @@ public void WriteClassMembers(IJsonClassGeneratorConfig config, StringBuilder sw
                 string classPropertyName = this.GetCSharpPascalCaseName(field.MemberName);
                 string propertyAttribute = config.GetCSharpJsonAttributeCode(field);
 
-                if( !first && (propertyAttribute.Length > 0 || config.ExamplesInDocumentation) )
+                // If we are using record types and this is not the first iteration, add a comma and newline to the previous line
+                // this is required because every line except the last should end with a comma
+                if (config.RecordTypes && !first)
+                {
+                    sw.Append($",{Environment.NewLine}");
```

Comment:
Why `sw.Append($",{Environment.NewLine}");` instead of `sw.AppendLine(",");` ?

Reply:
I'm not sure either, I'll update it.